public/data/ksaurabh4.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Kumar Saurabh",
+  "type": "personal",
+  "bio": "Full Stack Developer | Freelancer | Learner | Traveller",
+  "avatar": "https://github.com/ksaurabh4.png",
+  "links": [
+    {
+      "name": "My Portfolio website",
+      "url": "https://ksaurabh4.github.io/portfolio/",
+      "icon": "globe"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ksaurabh4",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ksaurabh4/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ksaurabh4",
+      "icon": "twitter"
+    },
+    {
+      "name": "Checkout my Blogs",
+      "url": "https://www.whynotgoogleit.com",
+      "icon": "globe"
+    },
+    {
+      "name": "My Motivational Youtube Channel",
+      "url": "https://youtube.com/thoughtcurator",
+      "icon": "youtube"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "June 2020",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": " https://www.linkedin.com/in/ksaurabh4/"
+    },
+    {
+      "title": "Started Opensource",
+      "date": "June 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Created Video screen in an open source project animeworldz",
+      "url": "Anime Streaming App"
```

Comment:
_Please **place a correct URL here** else **delete this line!**_ 


**Summary:** This code needs a Fix! 🛠

Reply:
Fixed